### PR TITLE
Learn by doing, Homework 3, 10/5/5: Initial Commit

### PR DESCRIPTION
**Pull request by nsackett**
[#104952774]

### CODE REVIEW

File: chapter_3/number_of_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+minute = 60
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: Would `seconds_per_minute` be more descriptive?

File: chapter_3/number_of_seconds.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+hour = 60 * 60
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: `seconds_per_hour`

File: chapter_3/number_of_seconds.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+day = 60 * 60 * 24
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: `seconds_per_day`

File: chapter_3/number_of_seconds.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+week = 60 * 60 * 24 * 7
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: `seconds_per_week`

File: chapter_3/number_of_seconds.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+puts 'There are ' + minute.to_s + ' seconds in a minute.'
+puts ''
+puts 'There are ' + (hour / minute).to_s + ' minutes in an hour.'
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: A variable `minutes_per_hour` would be preferable.

File: chapter_3/number_of_seconds.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+puts ''
+puts 'There are ' + (day / hour).to_s + ' hours in a day.'
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: `hours_per_day`

File: chapter_3/number_of_seconds.rb [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+puts ''
+puts 'There are ' + (week / day).to_s + ' days in a week.'
```
Comment (jasonnoble) [inline chapter_3/number_of_seconds.rb]: `days_per_week`

Comment (nsackett) [inline chapter_3/number_of_seconds.rb]: Modified to make variable names more descriptive.

**Pull request merged**